app/views/layouts/temp_home.html.erb

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
```

Comment:
Can you change this to HAML. There are online web sites that will mostly do it for you.